actionpack/lib/action_controller/api.rb

```diff
@@ -37,7 +37,7 @@ module ActionController
   # == Renders
   #
   # The default API Controller stack includes all renderers, which means you
-  # can use <tt>render :json</tt> and brothers freely in your controllers. Keep
+  # can use <tt>render :json</tt> and siblings freely in your controllers. Keep
```

Comment:
using family related names at all seems weird to me already. What do you think about using something like,

```suggestion
  # can use <tt>render :json</tt> and related methods freely in your controllers. Keep
```

I think `siblings` is also better, so this change is a 👍 from me either way.

Reply:
`siblings` is more correct here because it is talking about `render text:` which is not a new method.